packages/@sanity/portable-text-editor/src/editor/plugins/createWithPortableTextLists.ts

```diff
@@ -143,11 +143,14 @@ export function createWithPortableTextLists(types: PortableTextMemberSchemaTypes
       const selectedBlocks = [
         ...Editor.nodes(editor, {
           at: editor.selection,
-          match: (node) => editor.isListBlock(node) && node.listItem === listStyle,
```

Comment:
On the face of it, this seems identical, but `match` essentially filters all blocks in the selection, which then conceals whether any of these blocks are *not* of a list type